.eslintrc

```diff
@@ -16,10 +16,10 @@
     "complexity": [2, 55],
     "max-statements": [2, 115],
     "no-unreachable": 1,
-    "no-useless-escape": 0,
+    "no-console": 0,
```

Comment:
Yes, for now. `console` is used in the running code to output warning messages.

But this is from a practical viewpoint. I would be happiest if as many checks as possible are enabled for `eslint`. It's just that `console` is used a bit too often to casually fix this.